### PR TITLE
[main] Migrating to compose.foundation insets

### PR DESCRIPTION
**Pull request by astamato**

### CODE REVIEW

File: AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/home/MainActivity.kt [hunk 1]
```diff
@@ -36,17 +36,19 @@ class MainActivity : ComponentActivity() {
 
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
+        Paint().alpha= 3.4f
```
Comment (JolandaVerhoef) [inline AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/home/MainActivity.kt:39]: I don't think this should be here?

Comment (astamato) [reply AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/home/MainActivity.kt:39]: TY hmm no idea how that got there

File: AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt [hunk 2]
```diff
@@ -96,28 +98,22 @@ private fun ExploreList(
             }
         }
         item {
-            Spacer(modifier = Modifier.navigationBarsHeight())
+            Spacer(modifier = Modifier.windowInsetsBottomHeight(WindowInsets.navigationBars))
         }
```
Comment (bentrengrove) [inline AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/base/ExploreSection.kt:102]: Same comment as end, I think we should use contentPadding here instead

**Review by bentrengrove — APPROVED** [review]
Just 1 nit

**Pull request merged**